pesign-repackage.spec.in

```diff
@@ -217,6 +217,18 @@ rm "$cert.pub"
 popd
 /usr/lib/rpm/pesign/pesign-gen-repackage-spec @PESIGN_REPACKAGE_COMPRESS@ @PESIGN_LOAD_SPEC_MACROS@ \
 	--directory=%buildroot "${rpms[@]}"
+
+# For some reason in Fedora builds the directory structure is different from SUSE,
+# which breaks repacking. Copy the package content to the buildroot that is actually used.
+# Also all the usual tricks to disable the debug package fail, and the build fails due to
+# the 'Empty files file <...>/debugsourcefiles.list' error. Delete the specpart to bypass it.
+source /usr/lib/os-release || true
+if [ "${ID:-}" = "fedora" ] || [ "${ID_LIKE:-}" = "fedora" ]; then
```

Comment:
could this be done via `%if 0%{?fedora}?` instead or which other distribution are you trying to cover here?

Reply:
let me try that